firestore-bigquery-export/scripts/import/src/config.ts

```diff
@@ -197,3 +217,22 @@ export async function parseConfig(): Promise<CliConfig> {
     multiThreaded: multiThreaded,
   };
 }
+
+/**
+ *
+ * @param template - eg, regions/{regionId}/countries
+ * @param text - eg, regions/asia/countries
+ *
+ * @return - eg, { regionId: "asia" }
+ */
+export const resolveWildcardIds = (template: string, text: string) => {
+  const textSegments = text.split("/");
+  return template
+    .split("/")
+    .reduce((previousValue, currentValue, currentIndex) => {
+      if (currentValue.startsWith("{") && currentValue.endsWith("}")) {
```

Comment:
Is there are a reason for this if statement? Could this be `[oddvalue] = evenValue`?

Reply:
I'm not sure what you mean?

Reply:
Previously we had the values in the form of `path_params` but I see now we don't have those here! Approving